string cmd
float64[] joint_position
geometry_msgs/Twist pose
bool vaccum
